### ISSUE TITLE
Calculate rollover of new robot design

### ISSUE THREAD

**Issue by AuroTB** [RoBorregos/Robocup-Home#90]
Calculate force/momentum according to the robot's new points of center mass created by the arms/base.
- Rotating robot
- Moving robot

(idk plz edit this issue)

**Comment by AminMendez** [comment]
In order to calculate the roll over in certain circumstances first we need to calculate the center of gravity and inertia.

In case of crash or Unespected stop.
 
The emergency boton will stop the wheels inmediatly causing the robot transform its kinetic energy into potencial energy trying to rotate the robot. Using SolidWorks mass properties we can determine it with respect of the wheels. We can use the same point in case of a lower zone crash.

![Center of gravity](https://user-images.githubusercontent.com/47193853/100036247-7cc49f80-2dc5-11eb-94a5-f4455320c34a.JPG)
![Center of Gravity with respecto of the Wheel](https://user-images.githubusercontent.com/47193853/100037212-4851e300-2dc7-11eb-85a0-ea7a2f48a3ca.JPG)
![image](https://user-images.githubusercontent.com/47193853/100037508-e2199000-2dc7-11eb-9b00-b7143ed189f5.png)

In conclusion of the result, we can know that maximum velocity at which the robot can move with the actual structure, the arm at its hight position and stop without falling is 0.9 meters per second
